docs/oauth.md

```diff
@@ -96,6 +96,16 @@ Each OAuth config attribute is represented by an element in the XML, the element
 | accessTokenResponseMaps | Map<String, String> | Key value pair that maps an initial token request response attribute <value> to Tableau recognized attribute <key> | No | |
 | refreshTokenResponseMaps | Map<String, String> | Key value pair that maps an refresh token request response attribute <value> to Tableau recognized attribute <key> | Yes | If not defined will use accessTokenResponseMaps by default |
 
+Note: The keys in accessTokenResponseMaps and refreshTokenResponseMaps are tableau preserved field names and is defined in the followed table, the values are what your oauth provider returns in the raw response.
+| Name of the key | Optional/Required | Notes |
```

Comment:
Have a Required column for accessTokenResponseMaps and one for refreshTokenResponseMaps. I think that will read more easily.